packages/cactus-plugin-ledger-connector-iroha/src/main/json/openapi.json

```diff
@@ -173,9 +173,646 @@
+                "$ref": "#/components/schemas/SetAccountQuorumRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/AddAssetQuantityRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/SubtractAssetQuantityRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/TransferAssetRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetSignatoriesRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAccountRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAccountDetailRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAssetInfoRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAccountAssetsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/AddSignatoryRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/RemoveSignatoryRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/CreateRoleRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/AppendRoleRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/DetachRoleRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetRolePermissionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GrantPermissionRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/RevokePermissionRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetTransactionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetPendingTransactionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAccountTransactionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAccountAssetTransactionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetBlockRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/CallEngineRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetEngineReceiptsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/AddPeerRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/RemovePeerRequestParameters"
+              }
+            ]
```

Comment:
One more suggestion - order of references.
I see order of entries:
1. CreateAccountRequestParameters
2. SetAccountDetailRequestParameters
3. CompareAndSetAccountDetailRequestParameters
4. CreateAssetRequestParameters
5. CreateDomainRequestParameters
6. SetAccountQuorumRequestParameters
7. AddAssetQuantityRequestParameters
8. SubtractAssetQuantityRequestParameters
9. TransferAssetRequestParameters
10. GetSignatoriesRequestParameters
11. GetAccountRequestParameters
12. GetAccountDetailRequestParameters
13. GetAssetInfoRequestParameters
14. GetAccountAssetsRequestParameters
15. AddSignatoryRequestParameters
16. RemoveSignatoryRequestParameters
17. CreateRoleRequestParameters
18. AppendRoleRequestParameters
19. DetachRoleRequestParameters
20. GetRolePermissionsRequestParameters
21. GrantPermissionRequestParameters
22. RevokePermissionRequestParameters
23. GetTransactionsRequestParameters
24. GetPendingTransactionsRequestParameters
25. GetAccountTransactionsRequestParameters
26. GetAccountAssetTransactionsRequestParameters
27. GetBlockRequestParameters
28. CallEngineRequestParameters
29. GetEngineReceiptsRequestParameters
30. AddPeerRequestParameters
31. RemovePeerRequestParameters
But it looks like random order. I suggest: first commands then queries. Then if there is no specific order in Cactus's style I suggest to use the same order as in Iroha: https://iroha.readthedocs.io/en/develop/develop/api.html - it is alphabetically

Reply:
- [x] I see that it is sorted after update.